locales/de.json

```
@@ -268,7 +268,7 @@
     "Webinars": "Webinare",
     "DisableMFA": "MFA deaktivieren",
     "EnableMFA": "MFA aktivieren",
-    "TOTPSecret": "TOTP-Geheimnis",
+    "TOTPSecret": "TOTP-Einmalpasswort",
```

Hiermit sollte tatsächlich das Secret gemeint sein, woraus dann die Codes generiert werden. "Geheimnis" war also schon gar nicht so falsch, aber würde es auch einfach "TOTP-Secret" nennen.

```suggestion
    "TOTPSecret": "TOTP-Secret",
```